addons/l10n_es_edi_tbai/models/account_edi_format.py

```diff
@@ -110,7 +110,9 @@ def _l10n_es_tbai_post_invoice_edi(self, invoice):
             # - If called manually, then the user will see this error pop up when it triggers
             chain_head = invoice.company_id._get_l10n_es_tbai_last_posted_invoice()
             if chain_head and chain_head != invoice and not chain_head._l10n_es_tbai_is_in_chain():
-                raise UserError(f"TicketBAI: Cannot post invoice while chain head ({chain_head.name}) has not been posted")
```

Comment:
@agr-odoo We should not be raising in this case, but return an error as it could be run by a cron.

Reply:
Seems right, although I couldn't find an error log mechanism for the returned error